### DISCUSSION TITLE
1st gen thread ripper vs i9 9900k software encoding?

### DISCUSSION THREAD

**Discussion by 54x1** [cdgriffith/FastFlix#555] [Q&A]
hi just wondering which is better for software encoding 1st gen thread ripper or i9 9900k? 
Fastflix uses multi core and multi threads right?

**Comment by cdgriffith** [comment]
https://www.anandtech.com/show/11697/the-amd-ryzen-threadripper-1950x-and-1920x-review/10

The 9900k does about the same as the 7900X, and that trades blows with the 1950x

**Comment by 54x1** [comment]
but i need a mb that can handle 4080s, 2070s and 1070ti all running at the same time

**Comment by 54x1** [comment]
@cdgriffith oh for future reference which cpu is best for x266 or x265 software encoding

ryzen 9 7950x | ryzen 9 7950x3d | intel i9 14th gen | intel i9 13th gen
